content/docs/guides/ads_on_amp/ads_tips@es.md

```diff
@@ -0,0 +1,43 @@
+---
+$title : Tips para Anuncios sobre páginas AMP
+$order : 1
+---
+
+### Colocación & Controles 
```

Comment:
I noticed the formatting on these headings differ from the English version.  Intentional?